.github/workflows/docs.yml

```
@@ -0,0 +1,29 @@
+name: Docs
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
```

```suggestion
on:
  # * only trigger on pull request closed to specific branches
  # ref: https://github.community/t/trigger-workflow-only-on-pull-request-merge/17359/9
  pull_request:
    branches: [ master ]
    types: [ closed ]

jobs:
  docs:
    if: github.event.pull_request.merged == true ## only for PR merges
```

Trigger only on PR merges (GitHub has some build limits for the site)?

---

sure